src/components/TodosFilter.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useContext } from 'react';
+import { Filter } from '../types/Filter';
+import { TodosContext } from './TodosContext';
+
+export const TodosFilter: React.FC = () => {
+  const { filterType, setFilterType } = useContext(TodosContext);
+  const makeSetFilterType = (type: Filter) => (
+    () => setFilterType(type));
+
+  return (
+    <ul className="filters" data-cy="todosFilter">
+      <li>
+        <a
+          href="#/"
+          className={filterType === Filter.ALL ? 'selected' : ''}
```

Comment:
use classnames library for setting dyanmic classNames
https://www.npmjs.com/package/classnames